pulp_rpm/tests/functional/api/test_pulpimport.py

```diff
@@ -310,11 +310,156 @@ def test_clean_import(self):
         filenames = [f for f in list(self.export.output_file_info.keys()) if f.endswith("tar.gz")]
         importer = self._create_importer()
         self._post_export_cleanup()
-
+        existing_repos = self.repo_api.list().count
         # At this point we should be importing into a content-free-zone
         import_response = self.imports_api.create(importer.pulp_href, {"path": filenames[0]})
         task_group = monitor_task_group(import_response.task_group)
         self.assertEqual(len(self.import_repos) + 1, task_group.completed)
         for repo in self.import_repos:
             repo = self.repo_api.read(repo.pulp_href)
             self.assertEqual(f"{repo.pulp_href}versions/1/", repo.latest_version_href)
+        # We should have the same number of repositories, post-import, than however many we had
+        # post-removal of the export-repos
+        self.assertEqual(self.repo_api.list().count, existing_repos)
+
+
+class CreateMissingReposImportTestCase(PulpTestCase):
+    """
+     Tests for PulpImporter and create-missing-repos.
+
+     1. Create a regular and a dist-tree remote
+     2. Create a repo for each
```

Comment:
Do you think it would make sense to split this into two separate tests with a generic shared setup / test procedure?